en/docs/genai/tutorials/review-summarizer-natural-function.md

```
@@ -0,0 +1,326 @@
+
+## 4. Run and Test
+
+### Step 4.1 — Run the Service
+
+From the **Overview** page click **Run**. BI applies the `--experimental` flag and starts the service.
+
+![Integration Overview page with the Run button highlighted in the top-right toolbar. The design diagram shows httpDefaultListener connected to the /api/v1 service with a POST /analyze resource. A toast at the bottom right reads "WSO2 default model provider configuration values were added to t…".](/img/genai/develop/natural-functions/40-design-overview-run.png)
+
+The integrated terminal shows the build and run output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)